deployments/mainnet/configuration.json

```diff
@@ -0,0 +1,72 @@
+{
+  "symbol": "📈USDC",
+  "baseToken": "USDC",
+  "baseTokenAddress": "0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48",
+  "baseTokenPriceFeed": "0x8fFfFfd4AfB6115b954Bd326cbe7B4BA576818f6",
+  "borrowMin": "1000e6",
+  "storeFrontPriceFactor": 0.5,
+  "targetReserves": "5000000e6",
+  "rates": {
+    "supplyKink": 0.8,
+    "supplySlopeLow": 0.0325,
+    "supplySlopeHigh": 0.4,
+    "supplyBase": 0,
+    "borrowKink": 0.8,
+    "borrowSlopeLow": 0.035,
+    "borrowSlopeHigh": 0.25,
+    "borrowBase": 0.015
+  },
+  "tracking": {
+    "indexScale": "1e15",
+    "baseSupplySpeed": "0.000011574074074074073e15",
+    "baseBorrowSpeed": "0.0011458333333333333e15",
```

Comment:
I still think we should set non-zero supply speed, personally

Reply:
The current configuration is 1 and 99 COMP per day, but the launch doc says 0 and 50. Should I go and make those changes? I do agree that non-zero supply speed would be nice though.

Reply:
Yea I still think we should do non-zero, but I think 50 total is more final

Reply:
Ok left supply speed as non-zero and changed borrow speed to 50.

Reply:
Actually, setting supply speed as non-zero means people can start accruing rewards before supply caps have been increased from zero. I'm not sure if this is something we want, since I assume we don't want ppl to be accruing rewards until the governance proposal has been executed.

Reply:
Oh interesting, good point, bc people can supply base, but they can't borrow until supply caps are non-zero 🤔 I guess we could raise `baseMinForRewards` and lower it in the proposal also, though doubt we like that idea. I guess we don't really have any way of preventing people from supplying base until proposal is executed regardless. If the rewards are tiny, I'm not sure it matters, and maybe its even a good thing to get supply in...

Reply:
Probably a good argument to set rewards speeds in the proposal as well

Reply:
Yeah setting reward speeds in the proposal is a good option.

Reply:
I've set the initial reward speeds to 0.